blockball-api/src/main/java/com/github/shynixn/blockball/api/persistence/entity/BallMeta.kt

```diff
@@ -64,6 +64,12 @@ interface BallMeta {
      * Should the ball always bounce of walls?
      */
     var alwaysBouce: Boolean
+
```

Comment:
Is this relevant to the arena creator?

Reply:
Probably not. Would you tell me the reason we have to skip interaction checks? And how much amount of tick that you think is relevant?